pkg/protoc/wrapper.go

```diff
@@ -0,0 +1,118 @@
+package protoc
+
+import (
+	"context"
+	"os"
+	"path/filepath"
+
+	"github.com/izumin5210/clicontrib/pkg/clog"
+	"github.com/izumin5210/gex/pkg/tool"
+	"github.com/pkg/errors"
+	"github.com/spf13/afero"
+	"k8s.io/utils/exec"
+
+	"github.com/izumin5210/grapi/pkg/clui"
+	"github.com/izumin5210/grapi/pkg/grapicmd/util/fs"
+)
+
+type Wrapper interface {
+	Exec(context.Context) error
+}
+
+type wrapperImpl struct {
+	cfg      *Config
+	fs       afero.Fs
+	ui       clui.UI
+	execer   exec.Interface
+	toolRepo tool.Repository
+	rootDir  string
+	binDir   string
+}
+
+func NewWrapper(cfg *Config, fs afero.Fs, execer exec.Interface, ui clui.UI, toolRepo tool.Repository, rootDir, binDir string) Wrapper {
```

Comment:
**[golint]** <sub>reported by [reviewdog](https://github.com/haya14busa/reviewdog) :dog:</sub>
exported function NewWrapper should have comment or be unexported

Reply:
fixed in d26e797